web/e2e/page-objects/flows/manageFlows.ts

```diff
@@ -143,6 +143,11 @@ export class ManageFlows extends AppPage {
     return await link.click();
   }
 
+  async clickFlowName(flowname: string) {
+    let link = this.flowName(flowname);
+    return await browser.executeScript("arguments[0].click();", link);
+  }
+
```

Comment:
Did we run into focus issues like intended view is behind some other window. Is that why we are using executeScript.

Reply:
It seems to be more reliable instead of .click() method in some cases.